pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -152,23 +152,32 @@ def localized_image(path)
 
 def campaign_date(format)
   @country ||= hoc_detect_country
+  type = HOC_COUNTRIES[@country]['type'] || 'default'
+  language = HOC_COUNTRIES[@country]['default_language']
+  id = 'campaign_date_full'
 
   case format
   when "start-short"
-    return HOC_COUNTRIES[@country]['campaign_date_start_short']
+    id = 'campaign_date_start_short'
   when "start-long"
-    return HOC_COUNTRIES[@country]['campaign_date_start_long']
+    id = 'campaign_date_start_long'
   when "short"
-    return HOC_COUNTRIES[@country]['campaign_date_short']
+    id = 'campaign_date_short'
   when "full"
-    return HOC_COUNTRIES[@country]['campaign_date_full']
+    id = 'campaign_date_full'
   when "year"
-    return HOC_COUNTRIES[@country]['campaign_date_year']
+    id = 'campaign_date_year'
   when "full-year"
-    return HOC_COUNTRIES[@country]['campaign_date_full_year']
-  else
-    return HOC_COUNTRIES[@country]['campaign_date_full']
+    id = 'campaign_date_full_year'
+  end
+
+  if type == 'international'
+    id = "international_#{id}"
+  elsif type == 'europe'
+    id = "europe_#{id}"
   end
+
+  return HOC_I18N[language][id] || HOC_I18N['en'][id]
```

Comment:
What does `HOC_COUNTRIES` vs. `HOC_I18N` represent?

Reply:
`HOC_COUNTRIES` is the parsed `countries.json` file as an object. `HOC_I18N` is all the *.yml files (per language) for the hour of code translation variables--so the new variables I put in `en.yml` will be in `HOC_I18N`

Reply:
👍 thanks!